Move FP8 PRs from skip to cpu (x86)

chunyuan-w opened the pull request.

jbschlosser commented on 2.8.0/todo/result_cpu (x86).md:
```diff
@@ -28,12 +28,14 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- [CPU][Brgemm] add s8s8 GEMM microkernel API ([#154358](https://github.com/pytorch/pytorch/pull/154358))
```
do you mind removing the tags in brackets here? The release notes runbook recommends formatting titles without these

chunyuan-w replied:
Removed

jbschlosser reviewed (commented):
Thanks! If all the commits in `result_cpu (x86).md` are done, do you mind moving this file over to `2.8.0/done/...` as well?

chunyuan-w commented on 2.8.0/todo/result_cpu (x86).md:
```diff
@@ -28,12 +28,14 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- add s8s8 GEMM microkernel API ([#154358](https://github.com/pytorch/pytorch/pull/154358))
```
Fixed

chunyuan-w commented:
> Thanks! If all the commits in `result_cpu (x86).md` are done, do you mind moving this file over to `2.8.0/done/...` as well?

After second thoughts, I changed the category of the FP8 PRs in https://github.com/pytorch-labs/torch-release-notes/pull/5/commits/d7f4f0638a402d0152efd8fc36888d3b0bf15a41 and I've moved `result_cpu (x86).md` to `2.8.0/done/...`. Could you please help review again to see if it's good?

jbschlosser reviewed (approved):
looks good, thanks!
